src/views/League.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <Page>
+    <div class="league-page">
+      <iframe src="https://aeternity.com/league-iframe/index.html" width="100%" />
```

Comment:
this is a mirror copy of the league site, we can replace it once the issue with wordpress blocking iframes is resolved